packages/sources/metalsapi/src/config.ts

```diff
@@ -4,7 +4,7 @@ import { Config } from '@chainlink/types'
 export const NAME = 'METALSAPI'
 
 export const DEFAULT_BASE_URL = 'https://metals-api.com/api/'
-export const DEFAULT_ENDPOINT = 'convert'
+export const DEFAULT_ENDPOINT = 'latest'
```

Comment:
```suggestion
export const DEFAULT_ENDPOINT = 'convert'
```
We should leave this as is for backwards compatibility.

Reply:
Change didn't take :open_mouth: